.github/workflows/publish_sub_package.yml

```diff
@@ -4,14 +4,19 @@ on:
   push:
     branches:
       - main
+    pull_request:
```

Comment:
`pull_request` should at same level with `push`

Reply:
Fixed